### ISSUE TITLE
no-curly-component-invocation: False positive for block invocation with positional parameter

### ISSUE THREAD

**Issue by Turbo87** [ember-template-lint/ember-template-lint#1004]
Example: 

```hbs
{{#animated-if condition}}foo{{/animated-if}}
```

**Comment by rwjblue** [comment]
Yep, we fixed the `MustacheStatement` form but not the `BlockStatement` form. :+1: on fixing.
